k8s/calibrate/stage-calibrate-api.yaml

```diff
@@ -22,6 +22,30 @@ spec:
       labels:
         app: sta-calibrate
```

Comment:
could this (sta-calibrate) be a typo of sorts? Did we mean stage-calibrate?

Reply:
Hi @Baalmart , It's not a typo. Following the naming convention I found in place....instead of using `stage`, we use `sta`